html/components/card.stories.js

```diff
@@ -428,12 +433,17 @@ export const threeUpCards = () =>
             sprk-u-TextAlign--center
           "
         >
-          <a href="#nogo" class="sprk-u-AbsoluteCenter sprk-o-Stack__item">
+          <a
+            href="#nogo"
+            class="sprk-u-AbsoluteCenter sprk-o-Stack__item"
+            aria-label="Icon Title"
```

Comment:
Good catch! 

I don't think "Icon Title" is a good placeholder label here. This alt text should be describing the link, not the icon.

Reply:
In our example, the link is for the card with the title of "Icon Title". So it is describing the link in this case.

Reply:
What if we changed this placeholder text to "Learn More" so that it matches the other links in the card that go to the same place? From a screen reader perspective, these are the same link:

```
<a href="#nogo">
   Learn More
</a>
```

```
<a href="#nogo" aria-label="Learn More">
   <svg ...></svg>
</a>
```